Sources/SwiftKuery/Table.swift

```diff
@@ -101,34 +138,115 @@ open class Table: Buildable {
         return result
     }
     
-    /// Add alias to the table, i.e., implement the SQL AS operator.
-    ///
-    /// - Parameter newName: A String containing the alias for the table.
-    /// - Returns: A new Table instance with the alias.
+    /// Function to create a String representation of a `Table` instance for use in an SQL CREATE TABLE query.
```

Comment:
We've not addressed Enrique's comment here -I think this needs rewording, something along the lines: `Function that returns the SQL CREATE TABLE query as a String`